src/__test__/Utilities.test.ts

```diff
@@ -0,0 +1,279 @@
+  //     expect(error.code).toBe(ErrorCodes.UsageError);
+  //     expect(error.message).toContain('rn_invalid_type');
+  //     expect(error.debugCode).toBe('rn_invalid_type');
+  //     expect(error.debugMessage).toBe('font should be an object type.');
+  //   }
+  //   try {
+  //     Utilities.verifyFontType({
+  //       name: 'hei',
+  //     });
+  //   } catch (ex) {
+  //     const error:ErrorDetails = JSON.parse(ex.message);
+  //     expect(error.code).toBe(ErrorCodes.UsageError);
+  //     expect(error.message).toContain('rn_invalid_type');
+  //     expect(error.debugCode).toBe('rn_invalid_type');
+  //     expect(error.debugMessage).toContain('font.size');
+  //   }
+  //   try {
+  //     Utilities.verifyFontType({
+  //       size: 20,
+  //       name: 1,
+  //     });
+  //   } catch (ex) {
+  //     const error:ErrorDetails = JSON.parse(ex.message);
+  //     expect(error.code).toBe(ErrorCodes.UsageError);
+  //     expect(error.message).toContain('rn_invalid_type');
+  //     expect(error.debugCode).toBe('rn_invalid_type');
+  //     expect(error.debugMessage).toContain('font.name');
+  //   }
+  // });
+
+  it('verifyColorType pass with valid color types', () => {
+    Utilities.verifyColorType({
+      r: 14.3,
+      g: 0,
+      b: 255,
+      a: 2.3,
+    });
+    Utilities.verifyColorType({
+      r: -14.3,
+      g: 0,
+      b: 255,
+    });
+  });
+
+  // it('verifyColorType create error with invalid type', () => {
+  //   expect.assertions(12);
+  //   try {
+  //     Utilities.verifyColorType('string');
+  //   } catch (ex) {
+  //     const error:ErrorDetails = JSON.parse(ex.message);
+  //     expect(error.code).toBe(ErrorCodes.UsageError);
+  //     expect(error.message).toContain('rn_invalid_type');
+  //     expect(error.debugCode).toBe('rn_invalid_type');
+  //     expect(error.debugMessage).toBe('color should be an object type.');
+  //   }
+  //   try {
+  //     Utilities.verifyColorType({
+  //       g: 0,
+  //       b: 255,
+  //       a: 2.3,
+  //     });
+  //   } catch (ex) {
+  //     const error:ErrorDetails = JSON.parse(ex.message);
+  //     expect(error.code).toBe(ErrorCodes.UsageError);
+  //     expect(error.message).toContain('rn_invalid_type');
+  //     expect(error.debugCode).toBe('rn_invalid_type');
+  //     expect(error.debugMessage).toContain('value.r');
+  //   }
+  //   try {
+  //     Utilities.verifyColorType({
+  //       r: 1.3,
+  //       g: 'string',
+  //       b: 255,
+  //     });
+  //   } catch (ex) {
+  //     const error:ErrorDetails = JSON.parse(ex.message);
+  //     expect(error.code).toBe(ErrorCodes.UsageError);
+  //     expect(error.message).toContain('rn_invalid_type');
+  //     expect(error.debugCode).toBe('rn_invalid_type');
+  //     expect(error.debugMessage).toContain('value.g');
+  //   }
+  // });
+
+  it('verifyThemeType verifies all fields', () => {
+    // const spyVerifyFontType = jest.spyOn(Utilities, 'verifyFontType').mockImplementation();
```

Comment:
what are these commented lines?

Reply:
I will remove these. These test cases are now not feasible since we can not pass invalid parameters as they have now become now type-specific.